pkg/validation/internal/operatorhub.go

```diff
@@ -22,33 +22,55 @@ import (
 // the projects on OperatorHub.io.
 //
 // This validator will ensure that:
+//
 // - The annotations capabilities into the CSV has a valid option, which are:
-// * "Basic Install"
-// * "Seamless Upgrades"
-// * "Full Lifecycle"
-// * "Deep Insights"
-// * "Auto Pilot"
+//
+// * Basic Install
+//
+// * Seamless Upgrades
+//
+// * Full Lifecycle
+//
+// * Deep Insights
+//
+// * Auto Pilot
+//
```

Comment:
+1 on this added spacing since it helps separate "capabilities" above and the "categories" below.